crates/contracts/src/lib.rs

```diff
@@ -3,18 +3,20 @@ pub mod paths;
 pub mod vault;
 
 include!(concat!(env!("OUT_DIR"), "/BalancerV2Authorizer.rs"));
+include!(concat!(env!("OUT_DIR"), "/BalancerV2StablePool.rs"));
+include!(concat!(env!("OUT_DIR"), "/BalancerV2StablePoolFactory.rs"));
 include!(concat!(env!("OUT_DIR"), "/BalancerV2Vault.rs"));
 include!(concat!(env!("OUT_DIR"), "/BalancerV2WeightedPool.rs"));
-include!(concat!(env!("OUT_DIR"), "/BalancerV2StablePool.rs"));
 include!(concat!(
     env!("OUT_DIR"),
-    "/BalancerV2WeightedPoolFactory.rs"
+    "/BalancerV2WeightedPool2TokensFactory.rs"
 ));
 include!(concat!(
     env!("OUT_DIR"),
-    "/BalancerV2WeightedPool2TokensFactory.rs"
+    "/BalancerV2WeightedPoolFactory.rs"
 ));
-include!(concat!(env!("OUT_DIR"), "/BalancerV2StablePoolFactory.rs"));
```

Comment:
Were these just reordered for fun then? Doesn't appear to have been alphabetized...

Reply:
The are in alphabetical order now, they weren't before.